resources/views/sale_pos/partials/sale_line_details.blade.php

```diff
@@ -100,10 +109,17 @@
                 @if($sell_line->line_discount_type == 'percentage') ({{$sell_line->line_discount_amount}}%) @endif
             </td>
             <td>
+                @php
+                    $calTax = $sell_line->product->tax == 1 ? $subTotal - ($subTotal /1.16) : 0;
```

Comment:
why $sell_line->product->tax is compared with static value(1)? That is not understandable.

Reply:
$sell_line->product->tax == 1 is to determine wheather this product accept tax or it is zero tax product. 1= tax applicable, 2= zero tax

Reply:
Then use constants.

Reply:
constant added